### ISSUE TITLE
Jupyter Notebook Cells left arrow does not expand/collapse the cell

### ISSUE THREAD

**Issue by iemejia** [microsoft/vscode-jupyter#11449]
## Environment data

**VS Code Environment:**
Version: 1.72.0-insider
Commit: b9f7f85e90b4d51b38fcbf4232801ab04bd659cb
Date: 2022-09-23T08:15:07.186Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: Yes

**VS Code Extensions**
Jupyter Extension: v2022.9.1002681037 Pre-Release
Python Extension: v2022.15.12631011 Pre-Release
Azure Machine Learning - Remote: v0.19.2022092309 Pre-Release
Azure Data Lake Tools: v0.2.15

**Python Environment**
Conda: 4.13.0
Python: 3.9.13
Type of virtual environment: Conda + IPyKernel

**.net Environment**
.net: 6.0.401

**Operating System**
macOS Monterrey 12.6
Macbook Air (M1, 2020)

## Expected behaviour

Clicking in an arrow should expand/collapse the cell

## Actual behaviour

When you click in the left arrow of a cell it shows the debug options.

Maybe this is by design but given that the same symbol is used universally for code section compaction/expansion it is confusing. The only way to compact/expand a cell by double clicking it.

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1. Create a notebook, put multiple cells on it with 5 to 10 lines of code
2. Try to compact expand the cells using the left arrow.

Actual behavior, I need to double click the cell (hand mouse pointer) to expand it because clicking in the 'arrow' does not expand/collapse, it has a different behavior and shows a debug option

![reportbug-azuremlremote8](https://user-images.githubusercontent.com/79476/192235809-5e34ecc8-c21f-4938-b528-28e2a3e406c3.gif)

In the rest of VS Code the same icon of arrow is used for expanding/collapsing

![reportbug-azuremlremote7](https://user-images.githubusercontent.com/79476/192235841-3e412dcf-c65e-42d5-8328-4442978a8d0b.gif)

**Comment by amunger** [comment]
this is by design, but still pinging @roblourens to get your feedback

**Comment by roblourens** [comment]
Yeah the chevron icon is probably overloaded, but the "small chevron next to menu button" pattern is consistent with other places in vscode. I don't know if there's anything to do here but I'll send it to @daviddossett
